metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/GraphIndexUtils.java

```diff
@@ -0,0 +1,135 @@
+package com.linkedin.metadata.kafka.hook;
+
+import com.datahub.util.RecordUtils;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.data.schema.PathSpec;
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.metadata.graph.Edge;
+import com.linkedin.metadata.models.RelationshipFieldSpec;
+import com.linkedin.mxe.MetadataChangeLog;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Slf4j
+public class GraphIndexUtils {
+
+  private GraphIndexUtils() { }
+
+  @Nullable
+  private static List<Urn> getActorList(@Nullable final String path, @Nonnull final RecordTemplate aspect) {
+    List<Urn> actorList = null;
+    if (path != null) {
```

Comment:
I actually think this would read cleaner without excessive nesting. So something like

`if (path == null) {return null};`

`if (!value.isPresent() {return null;)`

Because then it becomes a bit more obvious in the cases this returns null. Same for the below functions

Reply:
for sure makes sense!

Reply:
Sweet!